apis/apis.json

```diff
@@ -509,19 +511,32 @@
     }
   },
 
+  {
+    "id": "Google.Cloud.Spanner.Common.V1",
+    "type": "other",
+    "targetFrameworks": "netstandard1.5;net45",
+    "version": "2.0.0-beta00",
+    "description": "Common resource names used by all Spanner V1 APIs",
```

Comment:
This should probably not refer to "resource names" anymore.

Reply:
I wondered about that. It's accurate for now - we can modify it if it ever contains more than resource names. (I've modified it for Bigtable to be common code. That PR update is coming soon.)

Reply:
Ok makes sense.